x-pack/plugins/security_solution/server/lib/detection_engine/signals/utils.ts

```diff
@@ -811,12 +809,26 @@ export const createTotalHitsFromSearchResult = ({
   searchResult,
 }: {
   searchResult: { hits: { total: number | { value: number } } };
-}): number => {
-  const totalHits =
-    typeof searchResult.hits.total === 'number'
-      ? searchResult.hits.total
-      : searchResult.hits.total.value;
-  return totalHits;
+}): number | undefined => {
+  if (typeof searchResult.hits.total === 'undefined') {
+    return undefined;
+  }
+  return typeof searchResult.hits.total === 'number'
+    ? searchResult.hits.total
+    : searchResult.hits.total.value;
+};
+
+const calculateTotal = (prev: SignalSearchResponse, next: SignalSearchResponse): number => {
+  if (
+    typeof createTotalHitsFromSearchResult({ searchResult: prev }) === 'number' &&
+    typeof createTotalHitsFromSearchResult({ searchResult: next }) === 'number'
+  ) {
+    return (
+      createTotalHitsFromSearchResult({ searchResult: prev })! +
+      createTotalHitsFromSearchResult({ searchResult: next })!
+    );
+  }
+  return -1;
```

Comment:
Nit: you could DRY this up a bit and prevent unnecessary invocations with some variables: 

```suggestion
const calculateTotal = (prev: SignalSearchResponse, next: SignalSearchResponse): number => {
  const prevTotalHits = createTotalHitsFromSearchResult({ searchResult: prev });
  const nextTotalHits = createTotalHitsFromSearchResult({ searchResult: next });
  if (typeof prevTotalHits === 'number' && typeof nextTotalHits === 'number') {
    return prevTotalHits + nextTotalHits;
  }
  return -1;
};
```